packages/reactivue/src/useSetup.ts

```diff
@@ -43,6 +49,38 @@ export function useSetup<State, Props = {}>(
 
   // trigger React re-render on data changes
   useEffect(() => {
+    /**
+     * Invalidate setup after hmr updates
+     */
+    if (process.env.NODE_ENV === 'development') {
```

Comment:
We could make this check outside of `useEffect`

Reply:
And we shall have a safe `typeof process !== 'undefined' && process.env.NODE_ENV` check.